portal/templates/bond/bond-details.njk

```diff
@@ -38,27 +39,26 @@
 
       <div class="govuk-grid-column-full">
 
-        <div class="govuk-form-group">
-
-          <div class="govuk-character-count" data-module="govuk-character-count" data-maxlength="150">
-            <div class="govuk-form-group">
-              <label class="govuk-label" for="bondIssuer">
-                Bond issuer (optional)
-              </label>
-              <span id="bond-issuer-hint" class="govuk-hint">
-                Only enter if Bond issuer differs from the bank
-              </span>
-
-              <div class="govuk-grid-row">
-                <div class="govuk-grid-column-one-half">
-                  <input class="govuk-input govuk-js-character-count" id="bondIssuer" name="bondIssuer" type="text" maxlength="150" aria-describedby="bond-issuer-hint" value="{{ bond.bondIssuer }}" data-cy="bond-issuer">
-                </div>
-              </div>
-            </div>
-
-            <span id="bond-issuer-info" class="govuk-hint govuk-character-count__message" aria-live="polite">
-              You can enter up to 150 characters
-            </span>
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-one-half">
+            {{ govukCharacterCount({
+              name: "bondIssuer",
+              id: "bondIssuer",
+              value: bond.bondIssuer,
+              maxlength: 150,
+              label: {
+                text: "Bond issuer (optional)"
+              },
+              hint: {
+                text: "Only enter if Bond issuer differs from the bank"
```

Comment:
do we need a seperate data-cy for the hint for testing?

Reply:
E2E test coverage is not great for this page. Have added an E2E test for the hint.